tsconfig.json

```diff
@@ -6,7 +6,7 @@
       "skipLibCheck": true,
       "module": "esnext",
       "moduleResolution": "node",
-      "target": "esnext",
+      "target": "es5",
```

Comment:
If we can, let's keep it `es2017`. If uglify-js does not play well, can we use uglify-es or some other minifier? If it's not possible, we probably need to keep code non-production and fix limit to 600k.